azerothcore/azerothcore-wotlk
# Fix(DB/Quest): Repurposed Technology

The corpse does not disappear after use, so players can abuse it using it multiple times
Insert the fix to resolve this issue.

TESTS PERFORMED:
mission accepted and cadaver now disappears

HOW TO TEST THE CHANGES:
take quest 12035 check that the corpse disappears

KNOWN ISSUES AND TODO LIST:
The corpse does not disappear after use, so players can abuse it using it multiple times

Target branch(es):
 [Master](https://github.com/azerothcore/azerothcore-wotlk/commit/c1fb6dd6a1fd64db0e7321add9c5bdf66c439ce1)

How to test AzerothCore PRs
When a PR is ready to be tested, it will be marked as [WAITING TO BE TESTED].

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
